CMS-Enterprise/mint-app
# [MINT-3240] FE work for Deleting Category from Action Menu

# MINT-3240

## Description

* Created `RemoveCategoryForm` component, test, copy, and mutations 
* Add `removeCategory` and `removeSubcategory` props to MTOContext and MTOModal 
* Add functions to ActionMenu to pop up remove category modal
* 👋🏼 



## How to test this change

1.  Navigate to MTO Matrix and hit the `...`
2. Click `Remove category` or `Remove sub-category`
3. Verify that modal matches figma 
4. hit the red button
5. verify that the category/subcategory selected is now gone

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
